### PR TITLE
Added bullet and missile data starfall functions

### PR DESCRIPTION
**Pull request by Alexandre425**
Added functions that return a copy of bullet and missile data tables.

### CODE REVIEW

File: lua/starfall/libs_sv/acffunctions.lua [hunk 1]
```diff
@@ -1950,6 +1963,58 @@ function ents_methods:acfRoundType()
...
+
+--- Returns a table containing the missile's data
+-- @server
+function ents_methods:acfMissileData()
+	CheckType(self, ents_metatable)
+
+	local This = unwrap(self)
+
+	if not IsACFEntity(This) then SF.Throw("Entity is not valid", 2) end
+	if RestrictInfo(This) then return "" end
```
Comment (TwistedTail) [inline lua/starfall/libs_sv/acffunctions.lua:1989]: Return type should be consistent

File: lua/starfall/libs_sv/acffunctions.lua [hunk 2]
```diff
@@ -1950,6 +1963,58 @@ function ents_methods:acfRoundType()
...
 
+--- Returns the BulletData table of the ammo in an ACF ammo crate
+-- @server
+function ents_methods:acfBulletData()
+	CheckType(self, ents_metatable)
+
+	local This = unwrap(self)
+
+	if not IsACFEntity(This) then SF.Throw("Entity is not valid", 2) end
+	if RestrictInfo(This) then return "" end
```
Comment (TwistedTail) [inline lua/starfall/libs_sv/acffunctions.lua:1974]: Return type should be consistent

File: lua/starfall/libs_sv/acffunctions.lua [hunk 3]
```diff
@@ -1950,6 +1963,58 @@ function ents_methods:acfRoundType()
...
+
+	local BulletData = This.BulletData
+
+	return BulletData and shallowCopy(BulletData) or {}
```
Comment (TwistedTail) [inline lua/starfall/libs_sv/acffunctions.lua:1978]: Does it not expose functions or anything?

**Review by TwistedTail — CHANGES_REQUESTED** [review]

File: lua/starfall/libs_sv/acffunctions.lua (hunk 3 above)
Comment (Alexandre425) [reply lua/starfall/libs_sv/acffunctions.lua:1978]: Not from my experience. Here is a sample output:

```
Id	=	AC
Crate	=	249
RoundVolume	=	49.32300466136
Efficiency	=	1
ProjMass	=	0.21641803472049
Diameter	=	2
ProjArea	=	3.1415926535898
ShovePower	=	0.2
Caliber	=	2
Type	=	AP
Tracer	=	0.3
PropLength	=	6.98
PropMass	=	0.020831900885954
Ricochet	=	60
DragCoef	=	0.0014516316339566
LimitVel	=	800
CartMass	=	0.23724993560645
Priority	=	Projectile
ProjLength	=	8.72
PropArea	=	3.1415926535898
MuzzleVel	=	454.92201677042
```

**Comment by Alexandre425** [comment]
I copied them from another function as a template and didn't may much attention to the checks, my bad. Should be alright now, unless you want me to change `shallowCopy`

**Review by TwistedTail — APPROVED** [review]

**Pull request merged**